wren-ai-service/src/pipelines/ask/indexing_pipeline.py

```diff
@@ -134,9 +200,39 @@ def _format(view: Dict[str, Any]) -> str:
         return [_format(view) for view in views]
 
 
+class Indexing(BasicPipeline):
+    def __init__(
+        self,
+        document_store: DocumentStore,
+        view_store: DocumentStore,
+    ) -> None:
+        pipe = Pipeline()
+        pipe.add_component("cleaner", DocumentCleaner([document_store, view_store]))
+        pipe.add_component("view_indexer", ViewIndexer(view_store))
+        pipe.add_component("converter", MDLToDDLConverter())
+        pipe.add_component(
```

Comment:
due to consistency, should we also have a writer for view_store here instead of embedding it inside ViewIndexer?

maybe the writer for document_store(should be renamed to ddl_store?) is called ddl_writer, and the writer for view_store is view_writer?

Reply:
after discussion, we'll split the view indexer into two components and rename these components for better understanding.